src/views/Account/AccountConfirm.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from "react";
+import { useAlert } from "react-alert";
+
+import { StringParam, useQueryParams } from 'use-query-params';
+
+import { BASE_URL } from "../../core/config";
+
+import { RouteComponentProps } from "react-router";
+import { TypedAccountConfirmMutation } from "./queries";
+
+import "./scss/index.scss";
+
+
+const AccountConfirm: React.FC<RouteComponentProps> = ({ history }) => {
+
+  const [query] = useQueryParams({
+    email: StringParam,
+    token: StringParam,
+  });
+
+  const alert = useAlert();
+
+  React.useEffect(() => {
+    this.accountManagerFn({
+      variables: {email: query.email, token: query.token},
+    }).then((result) => {
+      const errors = result.data.confirmAccount.errors;
+      {
```

Comment:
What is this block of code for (curly braces)?

Reply:
Redundant, didn't see it, will remove that shortly.